apps/docs/src/guide/contracts/variable-outputs.md

```diff
@@ -1,23 +1,27 @@
-<!-- NOTE: Review the relevance of this documentation page. The TypeScript SDK manages Output variables automatically, which may make the current content lack sufficient context. Consider providing a detailed explanation of how transactions work in a UTXO-based blockchain before discussing Output variables. This approach will ensure users have a better understanding of the topic and its importance. -->
-
 # Variable Outputs
 
-You may need to send funds to the transaction output in certain scenarios. Sway provides a method called `transfer_to_address(coins, asset_id, recipient)` that we can use for this purpose, which allows you to transfer a specific number of coins for a given asset to a recipient address.
+Sway includes robust functions for transferring assets to wallets and contracts.
+
+When using these transfer functions within your Sway projects, it is important to be aware that each call will require an [Output Variable](https://specs.fuel.network/master/tx-format/output.html#outputvariable) within the [Outputs](https://specs.fuel.network/master/tx-format/output.html) of the transaction.
+
+For instance, if a contract function calls a Sway transfer function 3 times, it will requires 3 Output Variables present within the list of outputs into your transaction.
```

Comment:
```suggestion
For instance, if a contract function calls a Sway transfer function 3 times, it will require 3 Output Variables present within the list of outputs in your transaction.
```